frontend/kesaseteli/employer/src/pages/_document.tsx

```diff
@@ -1,33 +1,3 @@
-import * as Sentry from '@sentry/browser';
-import Document, {
-  DocumentProps,
-  Head,
-  Html,
-  Main,
-  NextScript,
-} from 'next/document';
-import React from 'react';
+import ServerDocument from 'shared/server/ServerDocument';
 
-process.on('unhandledRejection', (err) => {
-  Sentry.captureException(err);
-});
-
-process.on('uncaughtException', (err) => {
-  Sentry.captureException(err);
-});
-
-class MyDocument extends Document<DocumentProps> {
-  render(): React.ReactElement {
-    return (
-      <Html>
-        <Head />
-        <body>
-          <Main />
-          <NextScript />
-        </body>
-      </Html>
-    );
-  }
-}
-
-export default MyDocument;
+export default ServerDocument;
```

Comment:
Even shorter
```suggestion
export { default as ServerDocument } from 'shared/server/ServerDocument';
```

Reply:
Eslint complains: `ESLint: Prefer default export.(import/prefer-default-export)` which is kinda incorrect in this case but can't bother to change this now.